chunk/chunk_store.go

```diff
@@ -414,6 +426,7 @@ func (c *AWSStore) calculateDynamoWrites(userID string, chunks []Chunk) (map[str
 		entries := 0
 		for _, bucket := range c.bigBuckets(chunk.From, chunk.Through) {
 			hashValue := hashValue(userID, bucket.bucket, metricName)
+			rowWrites.Observe(hashValue, uint32(len(chunk.Metric)))
```

Comment:
> is this likely to end up with us distributing across hash buckets in a way that's aligned to how we're distributing across dynamo?

Thats unknown, as we don't know how dynamodb distributes across partitions.

> If so, won't that make the measurements less useful?

Less useful maybe, but I still think quite useful - this will give us good information on the distribution of our write load within the hash space, and any massive outliers should show up.